src/frontend/screens/WebView/index.tsx

```diff
@@ -318,9 +292,10 @@ export default function WebView() {
     setShowLoginWarningFor(null)
   }
 
-  if (!preloadPath && isEpicLogin) {
-    return <></>
-  }
+  const userAgent =
+    startUrl === epicLoginUrl
+      ? 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) EpicGamesLauncher'
+      : 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) Chrome/200.0 HeroicGamesLauncher'
```

Comment:
Hmm, maybe we don't need those double UA tbh, I only added this one to try to fix some issues with random errors on epic login, server errors.

Reply:
I'm thinking it may be worth keeping for some browser content. Or in the future let users configure user agent strings for browser apps